### ISSUE TITLE
Lol keylogger

### ISSUE THREAD

**Issue by Vladislav873127** [Koolkat692/Orcus-RAT-Compiled#1]
his program create a keylogge in your system :D , if you want to get hacked download this prog!

**Comment by zribe** [comment]
yep! i installed it on a vm to test it. made lots of requests to some website...
